### DISCUSSION TITLE
Enable / Disable exposure to assistants on an entity level - Home Assistant Cloud

### DISCUSSION THREAD

**Discussion by AlexCherrypi** [frenck/spook#840] [Ideas]
I would love to disable / enable the exposure to Google Assistant, Alexa, and Assist on an entity level.

For example:

I have a heating blanket that I control via a light dimmer. 
Obviously, you don't need a heating banket during the summer.
That's why I unplug it and put it away.
Unfortunately, this also means it is unavailable to Home Assistant and  consequently  unavailable to Google Home.

So every time I try to turn my lights (or my whole home) on or off via Google Home, it tells me it can't reach my heating blanket.
This is very annoying.

I would like to specifcally disable exposure of my heating blanket entity to the assistants when it is not available (not reachable via Zigbee). This, unfortunately, is currently not possible with Spook.

**Comment by THuebner29** [comment]
I have a similar use case to disable some entities from reporting an error:

I use an Android TV, which is normally completely shut down and therefore not available for HA. But in an automation, I use the entity to turn off the amplifier at the same time when I turn it off. 

Every time I restart HA, Spook reports this entity as not available, which is true because the TV is switched off at this time.
